rest_framework_json_api/schemas/openapi.py

```diff
@@ -392,6 +393,9 @@ def _find_related_view(self, view_endpoints, related_serializer, parent_view):
 
     def _field_is_one_or_many(self, field, view):
         serializer = view.get_serializer()
+        if field not in serializer.fields:
```

Comment:
Explanation 3:
This was found out by testing, and I think it's because of dynamic fields.
For example, [here](https://github.com/django-json-api/django-rest-framework-json-api/blob/master/example/serializers.py#L91) we remove the "featured" field when the serializer is initialized, but it [exists](https://github.com/django-json-api/django-rest-framework-json-api/blob/master/rest_framework_json_api/schemas/openapi.py#L349) in the serializer class itself, and therefore being checked.